api.swagger.json

```diff
@@ -41381,6 +41381,177 @@
       ],
       "x-annotation-methodName": "withId"
     },
+    "/{projectKey}/states/key={key}": {
+      "get": {
+        "description": "Get State by Key",
+        "operationId": "Get State by Key",
+        "responses": {
+          "200": {
+            "$ref": "#/responses/trait:deprecatable:200"
+          },
+          "400": {
+            "$ref": "#/responses/trait:errorable:400"
+          },
+          "401": {
+            "$ref": "#/responses/trait:errorable:401"
+          },
+          "403": {
+            "$ref": "#/responses/trait:errorable:403"
+          },
+          "404": {
+            "$ref": "#/responses/trait:errorable:404"
+          },
+          "500": {
+            "$ref": "#/responses/trait:errorable:500"
+          },
+          "503": {
+            "$ref": "#/responses/trait:errorable:503"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/trait:expandable:expand"
+          }
+        ],
+        "security": [
+          {
+            "oauth_2_0": [
+              "manage_project:{projectKey}",
+              "view_orders:{projectKey}"
```

Comment:
probably you should add "view_states:{projectKey}", because "view_orders:{projectKey}" is the deprecated one